### ISSUE TITLE
breakage of install page on mobile

### ISSUE THREAD

**Issue by guylepage3** [blockstack/blockstack.org#290]
Install page breakage on mobile

<img width="504" alt="screenshot 2017-04-25 17 12 45" src="https://cloud.githubusercontent.com/assets/1711854/25408044/9fe656e0-29da-11e7-8537-e74115c7eba9.png">
<img width="539" alt="screenshot 2017-04-25 17 15 58" src="https://cloud.githubusercontent.com/assets/1711854/25408120/e88aea50-29da-11e7-9d4b-d1adec06d0dc.png">

**Comment by estermer** [comment]
#295

**Comment by estermer** [comment]
#297 

How's this?

<img width="317" alt="screen shot 2017-04-27 at 9 07 44 am" src="https://cloud.githubusercontent.com/assets/17515551/25484722/2440f068-2b29-11e7-94b5-fefcbce7ee6a.png">

**Comment by guylepage3** [comment]
Nice! looks good. 

Couple of adjustments.
1. why don't we remove the lines between the release titles and the buttons. ie. between "Developer alpha" and button.
2. let's make the buttons full width

**Comment by estermer** [comment]
Holy crap, I didn't realize you commented back on this so long ago haha

I can try and make those adjustments.

LOL I was wondering why there was nothing merged on this yet haha.

**Comment by estermer** [comment]
So I got rid of the border lines but I am unsure about making the buttons full width. There are too many containers forcing the button to be the width it currently is.

I also tried a few tricks to get the child element to extend beyond it's parent elements and it will only extend beyond it's immediate parent. so I am at sort of a loss as to how I could get them buttons full width. 

Maybe since the page is now legible we can add this to the current version and create another issue to improve the styling farther?
